format/README.md

```diff
@@ -0,0 +1,18 @@
+# GraphAr Format Specification
+
+This folder contains protocol definitions for the GraphAr format.
+
+## How to generate code
+
+### Prerequisites
+
+- [protoc](https://developers.google.com/protocol-buffers/docs/downloads)
+- [buf](https://buf.build/docs/installation)
```

Comment:
It is important that we need to use buf >= 1.32 because we are on v2. May we mention it here?

Reply:
Added the version tag, thanks~